### PR TITLE
[SCB-1407] Add new field when print provideCache log

### PR DESCRIPTION
**Pull request by GuoYL123**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java [hunk 1]
```diff
@@ -68,6 +68,10 @@ protected boolean doRegister() {
     microserviceInstance.getHealthCheck().setTimes(serviceRegistryConfig.getResendHeartBeatTimes());
 
     String instanceId = srClient.registerMicroserviceInstance(microserviceInstance);
+    MicroserviceInstance temIns = srClient.findServiceInstance(microserviceInstance.getServiceId(),instanceId);
+    if(temIns!=null){
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java]: Use the code formatter in etc folder to format code

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java [hunk 2]
```diff
@@ -68,6 +68,10 @@ protected boolean doRegister() {
...
+      microserviceInstance.setTimestamp(temIns.getTimestamp());
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java]: Does service-center already add timestamp property and which version？ And if it is neccessary to set timestamp information for the service itself (may be do not need set this property, or we'd better to keep itself reference from service center, not set a single property)?

Comment (wujimin) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java]: i remember SC provide this property in very very very old version
but better to confirm that.

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java (hunk 1 above)
Comment (GuoYL123) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java]: done

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java (hunk 2 above)
Comment (GuoYL123) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java]: it's always provide this property , java-chassis don't use it. we need to display the timestemp

**Pull request merged**